### PR TITLE
Update data collators and Add models

### PR DESCRIPTION
**Pull request by gimmaru**
## Title

- Update data collators and Add models

## Description

- Updated data collators to utilize sequence parallel in Oslo trainer
- Add models by referring to the transformers library

### CODE REVIEW

File: oslo/transformers/tasks/data_bart_pretraining.py [hunk 1]
```diff
@@ -109,35 +107,42 @@ def __call__(self, examples: List[Dict[str, Any]]) -> Dict[str, Any]:
...
             examples,
             return_tensors="pt",
-            pad_to_multiple_of=self.pad_to_multiple_of,
+            pad_to_multiple_of=self.local_world_size
+            if self.local_world_size > 1
+            else None,
         )
 
-        if self.pad_to_multiple_of is not None:
+        if self.local_world_size > 1:
```
Comment (hyunwoongko) [inline oslo/transformers/tasks/data_bart_pretraining.py:115]: let's make some utility functions for this. 
this can make reduce redundancy. :)

**Comment by hyunwoongko** [comment]
Please post a small PR a little more often. A large PR can make reviews difficult, which can make our code less stable.

**Comment by hyunwoongko** [comment]
Please let me know when you are done.

**Comment by gimmaru** [comment]
I'll tell you if there's no problem after conducting more tests.

**Pull request merged**